audit.py

```diff
@@ -85,68 +89,114 @@ def format_int(integer: Optional[int]) -> str:
     return "" if integer is None else str(integer)
 
 
-def format_timestamp(timestamp: str, now: datetime) -> str:
+def format_friendly_timestamp(timestamp: str, now: datetime) -> str:
+    """Add a user friendly relative time to a timestamp."""
     return f'{timestamp} ({timeago.format(datetime.fromisoformat(timestamp), now)})'
 
 
-def format_member(member) -> str:
+def format_generated_timestamp(dt: datetime) -> str:
+    """Return standard phrase for the date and time the report is generated"""
+    dt_as_text = dt.astimezone().strftime('%Y/%m/%d %H:%M:%S %Z')
```

Comment:
You may want to use "%c" for the locale’s appropriate date and time representation?
```suggestion
    dt_as_text = dt.astimezone().strftime('%c %Z')
```